polar-core/src/error.rs

```diff
@@ -86,13 +85,21 @@ impl PolarError {
         }
 
         // Augment ResourceBlock errors with relevant snippets of parsed Polar policy.
-        if let ErrorKind::Parse(ParseError::ResourceBlock {
+        if let ErrorKind::Validation(ValidationError::ResourceBlock {
             ref mut msg,
             ref ranges,
-            ..
+            ref loc,
         }) = self.kind
         {
             if let Some(source) = source {
+                let (row, column) = crate::lexer::loc_to_pos(&source.src, *loc);
+                self.context.replace(ErrorContext {
+                    source: source.clone(),
+                    row,
+                    column,
+                    include_location: false,
```

Comment:
This should be unnecessary. This will never be set for any of these error types.

Reply:
Removed in 82dd969